doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -232,6 +233,14 @@ Example of regular TLS logging:
       "issuerdn": "C=US, O=Google Inc, CN=Google Internet Authority G2"
   }
 
+Example of regular TLS logging for resumed sessions:
+
+::
+
+  "tls": {
+      "session": "resumed"
```

Comment:
I initially had it as `"session_resumed": true` but changed it to this variant following PR feedback from previous review rounds. I'd be happy to change it back (or to something else). Should I change it back?